python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cv.py

```diff
@@ -0,0 +1,282 @@
+"""
+Chaos cross-validation 
+----------------------
+"""
+
+# %%
+# In this example, we show how to perform the cross-validation of the Ishigami model using polynomial chaos expansion.
+# More precisely, we use the methods suggested in [Müller & Guido, 2016], chapter 5, page 257.
```

Comment:
Links to the bibliography are better (this book is cited several times in the example). That way there is no need to have a "References" section in the example.